### DISCUSSION TITLE
DLLs missing product version

### DISCUSSION THREAD

**Discussion by whgibbo** [aws/aws-sdk-cpp#2887] [Q&A] [answered]
Was curious if  there are any plans to generate RC files for windows DLLs to include the product information
- Company name
- Product Version

**Comment by jmklix** [comment]
We don't have any plans to generate RC files for this sdk. What use case are you looking to use them for?

**Reply by whgibbo** [reply]
@jmklix it would help determine which version is deployed.

**Reply by jmklix** [reply, marked as answer]
Thanks for this suggestion and your feedback. I've made a feature request to track the progress [here](https://github.com/aws/aws-sdk-cpp/issues/2902). But I don't have a timeline for when this might get added
